.gitattributes

```diff
@@ -10,3 +10,6 @@ doc/cli.markdown text eol=lf
 # crlf for the eol conversion test files
 tests/test-data/projects/docker-compose/basic/service2/file2-crlf.sh eol=crlf
 tests/test-data/projects/no-docker-compose/basic/src/windows-crlf.sh eol=crlf
+
+# Prevent auto merging of the npm-shrinkwrap.json file: see notes in CONTRIBUTING.md
+/npm-shrinkwrap.json merge=binary
```

Comment:
@Page-, given your careful tidying up of dependencies in the `npm-shrinkwrap.json` file (#1645), I thought we should try and protect it from merge madness. :-)  I got inspiration from [this post](https://blog.scottlogic.com/2018/02/13/npm-shrikwrap-merges-and-conflicts.html), but I am not really super sure that this is the best approach. See also notes added to CONTRIBUTING.md. Thoughts?

Reply:
I think it makes sense to avoid auto-merges for the shrinkwrap since they could be fairly dangerous, especially as npm doesn't seem to care if the shrinkwrap makes sense or not and just installs it regardless (see the junk in `net-keepalive`s shrinkwrap that gets installed even though it's not in the package.json for the module)